File: Programs/EvenOdd/EvenOdd_modulus/Python/evenOdd_modulus.py

def evenOddCheck_modulus(eO_num):
    print("Method : evenOddCheck_modulus()")

    if (eO_num % 2) == 0:
        print(eO_num, "is Even")
    else:
        print(eO_num, "is Odd\n")

evenOdd = int(input("Enter a number to check if the value is Even or Odd : "))
evenOddCheck_modulus(evenOdd)

File: Programs/EvenOdd/EvenOdd_Modulus/Cpp/evenOdd_modulus.cpp
#include <iostream>

using namespace std;

void evenOddCheck_modulus(int);

int main()
{
    int evenOdd;

    cout << "Enter a number to check if the value is Even or Odd : " << endl;
    cin >> evenOdd;

    evenOddCheck_modulus(evenOdd);

    return 0;
}

void evenOddCheck_modulus(int eO_num)
{
    cout << "Method : evenOddCheck_modulus()" << endl;

    if(eO_num % 2 == 0)
    {
        cout << eO_num << " is Even"  << endl;
    }
    else
    {
        cout << eO_num << " is Odd" << endl;
    }

    return;
}